src/main/java/net/minecraftforge/world/WorldCapabilityData.java

```diff
@@ -0,0 +1,58 @@
+package net.minecraftforge.world;
```

Comment:
Found =/
I'm planning to propose some hooks with world, but anyway it should be refactored later(not now)... okay.